bokehjs/gulp/tasks/develop.coffee

```diff
@@ -8,8 +8,7 @@ subTasks = ["develop:install-js"]
 gulp.task "develop", subTasks, ->
 
 gulp.task "develop:watch", ["develop"], ->
-  gulp.watch "#{paths.buildDir.js}/*.js", ["develop:install-js"]
-  gulp.watch "#{paths.buildDir.css}/*css", ["develop:install-js"]
+  gulp.watch "#{paths.coffee.watchSources}", ["develop:install-js"]
```

Comment:
These should only trigger after successful builds.  With this change, this triggers whenever the `.coffee` files change which would trigger another install before they've been built.